cmp/filter/FilterChooserModel.js

```diff
@@ -136,12 +151,17 @@ export class FilterChooserModel extends HoistModel {
         this.setValue(value);
         this.setFavorites(favorites);
 
-        if (targetStore) {
+        if (target) {
```

Comment:
given the two-way binding here, should we be renaming this argument to `bind` rather than `target`?

Reply:
I like this idea - less misleading now that we're doing two-way binding, and we're already in the business of doing a breaking rename here. Will do the rename. 👍